WooCommerce/WooCommerceTests/Yosemite/InAppPurchaseStoreTests.swift

```diff
@@ -62,6 +62,10 @@ final class InAppPurchaseStoreTests: XCTestCase {
         XCTAssertTrue(result)
     }
 
+    /// Skipped
+    /// Setting storefront value in Xcode 16 in one test no longer resets in other tests, even when resetToDefaultState is called
+    /// Reason unknown
+    /// https://forums.developer.apple.com/forums/thread/764937
```

Comment:
Thank you for adding the link.

Following up on my suggestion above regarding `XCTSkip`, one could use a similar approach as 

https://github.com/woocommerce/woocommerce-ios/blob/38a1190bd9e74901208b6c3d92e80e0fa5711722/WooCommerce/WooCommerceTests/POS/ViewModels/TotalsViewModelTests.swift#L215

to print this info in the test report.

Maybe also DRY via 

```swift
func skipBecauseOfStoreFrontXcode16Issue() throws {
    try XCTSkip(true, "...")
}